tests/riotboot_flashwrite/tests/01-run.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import subprocess
+import sys
+
+from testrunner import run
+
+
+USE_ETHOS = int(os.getenv("USE_ETHOS", "1"))
+TAP = os.getenv("TAP", "riot0")
+BINDIR = os.getenv("BINDIR")
+
+
+def wait_for_update(child):
+    return child.expect([r"riotboot_flashwrite: processing bytes (\d+)-(\d+)",
+                         "riotboot_flashwrite: riotboot flashing "
+                         "completed successfully"],
+                        timeout=10)
+
+
+def make_notify(client_url, slot, version):
+    cmd = [
+        "coap-client",
+        "-m",
+        "post",
+        "coap://{}/flashwrite".format(client_url),
+        "-f",
+        "tests_riotboot_flashwrite-slot{}.{}.riot.bin".format(slot, version),
+        "-b",
+        "64",
+    ]
+    subprocess.Popen(cmd, cwd=BINDIR)
```

Comment:
Why not :
```suggestion
    subprocess.call(cmd, cwd=BINDIR)
```
?
with eventually the use of an assert.